composer.json

```diff
@@ -58,10 +59,10 @@
     "symfony/twig-bundle": "@stable",
     "symfony/validator": "@stable",
     "symfony/web-link": "@stable",
-    "symfony/yaml": "@stable"
+    "symfony/yaml": "@stable",
+    "ext-xmlwriter": "*"
```

Comment:
Feels like this should be up with the other `ext-*` libs? Not sure if this is why, but I'm getting 
> Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.

on install.

Reply:
good catch. i put the `ext-xmlwriter` dependency where it belongs and updated the lock file.